app/assets/stylesheets/partials/_currency_input.scss

```diff
@@ -0,0 +1,9 @@
+
+span.currency-prefix {
```

Comment:
in scss you could write this as 
```
.currency-prefix {
  span {
    // code
  }
  input {
    // code
  }
}
```

Reply:
Doesn't seem to work for some reason.  I'll leave as-is if that's ok